README.md

```diff
@@ -18,6 +18,7 @@ Looking for something to build? Check out [the suggestions tracker][suggestions]
   - [JSON](#json)
   - [Parsing](#parsing)
   - [Templating](#templating)
+  - [Unique IDs](#unique-ids)
```

Comment:
How about a text section? Like Elixir's list? https://github.com/h4cc/awesome-elixir#text-and-numbers

Otherwise I suspect this section will never have another member.

Reply:
Perfect I was worried about the same thing